pkg/agent/flowexporter/utils.go

```diff
@@ -53,6 +53,17 @@ func IsConnectionDying(conn *Connection) bool {
 	return false
 }
 
+// checkConntrackConnActive returns true if there are changes in connection's stats or
+// TCP state, representing the connection is active.
```

Comment:
s/representing the connection is active./indicating that the connection is active.